sdk/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -43,8 +43,8 @@ public void toStream(@NonNull JsonStream writer) throws IOException {
 
         writer.name("id").value(packageName);
         writer.name("buildUUID").value(config.getBuildUUID());
-        writer.name("duration").value(getDuration());
-        writer.name("durationInForeground").value(sessionTracker.getDurationInForeground(System.currentTimeMillis()));
+        writer.name("duration").value(getDurationMs()/1000);
+        writer.name("durationInForeground").value(sessionTracker.getDurationInForegroundMs(System.currentTimeMillis())/1000);
```

Comment:
This is not introduced in this changeset, but these values are different if the session is delivered immediately vs retried after a long attempted network request.